tower-build-cli/tower-build

```diff
@@ -26,18 +25,13 @@ def parse_arguments():
         action='store_true',
         default=False
     )
-    parser.add_argument(
-        '--builds-dir',
-        help="""Directory containing builds necessary to build an image.""",
```

Comment:
What does `"builds"` mean here?

Reply:
it's a legacy.. initially the folder contained the NX builds.. now only the TowerOS-Host image. I removed this flag which no longer really makes sense now.